client_internals_test_utils.go

```diff
@@ -54,6 +54,10 @@ func (ci *ClientInternal) SerializationService() *serialization.Service {
 	return ci.client.ic.SerializationService
 }
 
+func (ci *ClientInternal) ProxyManagerProxies() map[string]interface{} {
```

Comment:
How about renaming this function to just `Proxies`?

Reply:
Resolved, [0042502](https://github.com/hazelcast/hazelcast-go-client/pull/772/commits/0042502d165af4bbfd57b99474f816dd147c4599).